turnpoints/hllstr.json

```diff
@@ -1675,11 +1682,11 @@
       "desc": "9/27 22D"
     },
     {
-      "name": "Jolon",
+      "name": "xJolon",
       "lat": 35.962783,
       "lon": -121.1075,
       "elev": 304.8,
-      "desc": "26D",
+      "desc": "26D could not see a runway as of 6/5/2022",
       "landable": {}
```

Comment:
Did you want to mark this unlandable? If so, you can remove the line `landable: {}`

Reply:
I had left it since there are landable fields nearby, but it is probably better to remove the landable status so I've done that.  If you want to dive into the Jolon valley without knowing you can land there, go for it.